desktop/src/player.rs

```diff
@@ -224,6 +224,33 @@ impl ActivePlayer {
                     "DejaVu Sans Mono".into(),
                 ],
             );
+            player_lock.set_default_font(
+                DefaultFont::JapaneseGothic,
+                vec![
+                    "ヒラギノ角ゴ Pro W3".into(), // Mac with Japanese environment
+                    "MS UI Gothic".into(),        // Windows
+                    "Noto Sans CJK JP".into(),    // Linux
+                    "Arial Unicode MS".into(),    // Mac fallback
+                ],
+            );
+            player_lock.set_default_font(
+                DefaultFont::JapaneseGothicMono,
+                vec![
+                    "Osaka－等幅".into(),      // Mac with Japanese environment
+                    "MS UI Gothic".into(),     // Windows
+                    "Noto Sans CJK JP".into(), // Linux
+                    "Arial Unicode MS".into(), // Mac fallback
+                ],
+            );
+            player_lock.set_default_font(
+                DefaultFont::JapaneseMincho,
+                vec![
+                    "ヒラギノ明朝 Pro W3".into(), // Mac with Japanese environment
+                    "MS UI Gothic".into(),        // Windows
```

Comment:
https://learn.microsoft.com/en-us/typography/font-list/ms-pmincho maybe?

Reply:
Again it's the question of do we want to be Flash accurate

I'd like to see if a Japanese install of Flash Player on Windows gives different fonts than I'm seeing